Trading/Mode/daily_trading_mode/daily_trading.py

```diff
@@ -166,10 +168,14 @@ def _get_limit_price_from_risk(self, eval_note):
     and self.STOP_LOSS_ORDER_MAX_PERCENT
     """
 
-    def _get_stop_price_from_risk(self):
-        factor = self.STOP_LOSS_ORDER_MAX_PERCENT - (self.trader.risk * self.STOP_LOSS_ORDER_ATTENUATION)
-        return trading_modes.check_factor(self.STOP_LOSS_ORDER_MIN_PERCENT, self.STOP_LOSS_ORDER_MAX_PERCENT,
-                                          factor)
+    def _get_stop_price_from_risk(self, is_long):
+        max_percent = self.STOP_LOSS_ORDER_MAX_PERCENT if is_long \
+            else 2 * trading_constants.ONE - self.STOP_LOSS_ORDER_MIN_PERCENT
+        min_percent = self.STOP_LOSS_ORDER_MIN_PERCENT if is_long \
+            else 2 * trading_constants.ONE - self.STOP_LOSS_ORDER_MAX_PERCENT
```

Comment:
What is this 2 for?

Reply:
mutliplier should be:
0.XX for stop prices bellow the given price
1.XX for stop prices above the given price

considering our multiplier is 0.95 to 0.99, to get 1.01 to 1.05 we need to use 2 - multiplier when generating a factor >1